src/wfutil.c

```diff
@@ -1417,7 +1417,7 @@ MyMessageBox(HWND hwnd, DWORD idTitle, DWORD idMessage, DWORD wStyle)
 DWORD
 ExecProgram(LPTSTR lpPath, LPTSTR lpParms, LPTSTR lpDir, BOOL bLoadIt, BOOL bRunAs)
 {
-  DWORD          ret;
+  DWORD_PTR     ret;
   INT           iCurCount;

```

Comment:
similar to another comment; why DWORD_PTR?

Reply:
It's the same case (return value of ShellExecuteEx as a pointer to be interpreted as an integer where >32 is success.)